config/database.yml

```diff
@@ -16,3 +11,11 @@ production:
   host: <%= ENV['POSTGRES_HOST'] %>
   port: 5432
   pool: 50
+
+development:
+  adapter: postgresql
+  host: <%= ENV['POSTGRES_PORT_5432_TCP_ADDR'] %>
```

Comment:
This should probably revert back to the previous defaults when not in docker, for the sake of not destroying in-use development environments or causing trouble for people not using docker.  Might be achievable via defaults in `.env`?